Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceProcess;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenVpn
+{ 
+    internal class Program
+    {
+        public static int Main(string[] args)
+        {
+            if (args.Length == 0)
```

Comment:
The command line args could be more like with the interactive service to make life easier for the uninitiated. Especially the lack of a command line help was noticed and deemed missing by me.

Reply:
I think we already covered it - when you run it as is, it just works as a standalone app which talks to interactive service (like GUI) - this part comes in follow-up commit. If you want to install/run service, then sc is a way to go.